packages/core/styles/filters.scss

```diff
@@ -1,11 +1,4 @@
 .filters-section {
-  &__wrapper {
```

Comment:
Isn't this needed when titles are above filters? I'm not seeing any space above filters now.

Reply:
Can you screenshot what you're seeing? In Pages/Dashboard/Regressions API Filters Map it looks like there's sufficient space?